icu4c/source/configure.ac

```diff
@@ -147,10 +147,10 @@ AC_SUBST(ENABLE_RELEASE)
 : ${CXXFLAGS=""}
 
 # Checks for compilers
-AC_PROG_CC([clang gcc cc c99 c89 xlc_r xlc cl.exe icc])
+AC_PROG_CC([cc clang gcc c99 c89 xlc_r xlc cl.exe icc])
 # Make sure that we try clang++ first, which provides C++17 support.
 # The g++ compiler is less likely to support C++17.
```

Comment:
You'll need to also update this comment.

Reply:
Hmmm, should I remove the comment entirely?

Reply:
I think that's a good decision, that comment originates from a time when GCC didn't have sufficient C++11 support and it's no longer relevant.